FirebaseFunctions/Tests/ObjCIntegration/ObjCAPITests.m

```diff
@@ -34,8 +34,8 @@ - (void)apis {
   func = [FIRFunctions functionsForApp:app customDomain:@"my-domain"];
 
   NSURL *url = [NSURL URLWithString:@"http://localhost:5050/project/location/name"];
-  FIRHTTPSCallable *callable = [func HTTPSCallableWithURL:url];
```

Comment:
These two lines should continue to compile.

Reply:
Fixed in latest batch of commits.